pkg/network/cloud/transport.go

```diff
@@ -0,0 +1,46 @@
+package cloud
```

Comment:
Is there a specific reason for this file being named `transport`?

Reply:
It handles operations on the transport layer. Its a well-known convention in package oriented design in Golang.